scenario/src/ErrorReporter.ts

```diff
@@ -49,6 +56,14 @@ class CTokenErrorReporterType implements ErrorReporter {
 
     return detail.toString();
   }
+
+  getEncodedCustomError(errorName: string, args: unknown[]): string | null {
+    try {
+      return TokenErr.CustomErrors.encodeErrorResult(errorName, args)
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/cloudwalk/compound-protocol/pullRequest?prid=9803552)
```suggestion
      return TokenErr.CustomErrors.encodeErrorResult(errorName, args);
```